### ISSUE TITLE
New language: Tagalog

### ISSUE THREAD

**Issue by notkenzotayko** [squidfunk/mkdocs-material#4218]
### Contribution guidelines

- [x] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### Language availability

- [X] I've checked the [list of available languages](https://bit.ly/33vFDD0)

### Translation

```jinja
{% macro t(key) %}{{ {
  "language": "tl",
  "direction": "ltr",
  "clipboard.copy": "Kopyahin sa clipboard",
  "clipboard.copied": "Nakopya mula sa clipboard",
  "edit.link.title": "I-edit ang pahinang ito",
  "footer.previous": "Nakaraan",
  "footer.next": "Susunod",
  "footer.title": "Lagdang Pangwakas",
  "header.title": "Pamuhatan",
  "meta.comments": "Mga Komento",
  "meta.source": "Pinagmulan",
  "nav.title": "Nabigasyon",
  "search.placeholder": "Hanapin",
  "search.share": "Ibahagi",
  "search.reset": "Tanggalin",
  "search.result.initializer": "Sinisimulan ang paghahanap",
  "search.result.placeholder": "Mag-type upang simulan ang paghahanap",
  "search.result.none": "Walang nahanap na dokumento",
  "search.result.one": "1 magkatugmang dokumento",
  "search.result.other": "# magkatugmang mga dokumento",
  "search.result.more.one": "1 meron sa pahina na ito",
  "search.result.more.other": "# meron sa pahina na ito",
  "search.result.term.missing": "Nawawala",
  "search.title": "Hanapin",
  "select.language.title": "Pumili ng lenguwahe",
  "select.version.title": "Pumili ng bersyon",
  "skip.link.title": "I-skip tungo sa nilalaman",
  "source.link.title": "Pumunta sa repository",
  "source.file.date.updated": "Huling update",
  "source.file.date.created": "Nagawa",
  "tabs.title": "Mga tala",
  "toc.title": "Talaan ng nilalaman",
  "top.title": "Bumalik sa taas"
}[key] }}{% endmacro %}
```

**Comment by squidfunk** [comment]
Thanks! Added in 5f360d277.

**Comment by squidfunk** [comment]
Released as part of 8.4.0.

**Comment by notkenzotayko** [comment]
Thanks, Martin. Cheers!
